<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QWidget" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>域名管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #EEF3F8, stop:1 #D7E6F5);
}

QLineEdit {
    background-color: #FFFFFF;
    border: 2px solid #C9D6E8;
    border-radius: 8px;
    padding: 8px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 2px solid rgb(145, 168, 235);
}

QLineEdit:placeholder {
    border-color: #4c65af;
    outline: none;
}

QLineEdit:focus {
    border: 2px solid #4c65af;
    outline: none;
}

QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #1f49a9, stop:1 #4A8EC9);
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    border-radius: 8px;
    border: none;
}

QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #4c65af, stop:1 #3A78B5);
}

QPushButton:pressed {
    background-color: #3A78B5;
}

QLabel {
    color: #33475B;
    font-size: 16px;
    font-weight: bold;
    padding: 4px;
}

QComboBox {
    font-family: &quot;PingFang SC&quot;, &quot;Helvetica Neue&quot;, Arial, &quot;Hiragino Sans GB&quot;, &quot;Microsoft YaHei UI&quot;, &quot;Microsoft YaHei&quot;, SimSun, sans-serif;
    font-size: 16px;
    color: #333333; 
    border: 1px solid #e0e0e0;

    background-color: #ffffff; 
}

QComboBox:focus {
    border-color: #0078d7;
    border: none;
    outline: none;
}

#label{
    font-size:20px;
}

#usernameshow {
    font-size: 24px;
    font-weight: bold;
    color: #2B5D7C;
    text-align: left;
    background: transparent;
    border: none;  /* 移除边框 */
    outline: none; /* 移除获得焦点时的轮廓 */
}

#usernameshow:hover {
    font-size: 25px;
    color:rgb(124, 43, 59);
}


#helloEdit{
    font-size: 24px;
    font-weight: bold;
    color: #2B5D7C;
    text-align: center;
    background: transparent;
    border: none;  /* 移除边框 */
    outline: none; /* 移除获得焦点时的轮廓 */
}
#searchButton {
    font: inherit;
    text-transform: none;
    white-space: nowrap;
    line-height: 1;
    text-align: center;
    font-size: 18px;
    color: #fff;
    text-decoration: inherit;
    background-color: #1f49a9; /* 自定义背景颜色 */
    border: none;
    border-radius: 8px;
    font-family: &quot;Microsoft YaHei&quot;, Arial, sans-serif;
}

#searchButton:hover {
    background-color:rgb(39, 86, 197); /* 悬浮状态时的背景颜色 */
}

#searchButton:focus {
    outline: none; /* 去掉焦点时的边框 */
}


   </string>
  </property>
  <widget class="QFrame" name="horizontalFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1059</width>
     <height>700</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>700</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>700</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>720</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>390</y>
         <width>171</width>
         <height>261</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>261</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="infoButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>信息修改</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="quitButton">
            <property name="text">
             <string>退出登录</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="logo">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>141</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:url(:/images/logo.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>850</width>
        <height>700</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1080</width>
        <height>720</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="maximumSize">
          <size>
           <width>1080</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QLineEdit" name="helloEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>131</width>
            <height>51</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="text">
           <string>欢迎您！</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="usernameshow">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>0</y>
            <width>430</width>
            <height>70</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>720</x>
            <y>10</y>
            <width>81</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>600</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QFrame" name="createFrame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">QFrame {
    border-top: none;         /* 不显示上边框 */
    border-left: none;        /* 不显示左边框 */
    border-right: none;       /* 不显示右边框 */
    border-bottom: none;  /* 设置下边框为 3 像素黑色实线 */
}
</string>
               </property>
               <property name="text">
                <string>创建域名：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="domainInput">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>44</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>370</width>
                 <height>300</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>请输入关键域名，如neu</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="domainChoose">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>44</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>.com</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>.edu</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>.top</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>.cn</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="categoryChoose">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>44</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>44</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>类别</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>商业</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>组织</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>网络</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>教育</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>政府</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>科技</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>商店</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>博客</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>法律</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>健康</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>应用程序</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>开发者</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>新闻</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>音乐</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>其他</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="searchButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>44</height>
                </size>
               </property>
               <property name="text">
                <string>提交</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="horizontalFrame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>55</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>250</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="findEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>请输入要查找的域名</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>200</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="searchButton_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>44</height>
                </size>
               </property>
               <property name="text">
                <string>查找</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="minimumSize">
             <size>
              <width>806</width>
              <height>460</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>460</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
